bff/test/Bff.Tests/Endpoints/Management/LoginEndpointTests.cs

```diff
@@ -28,6 +28,86 @@ public async Task login_should_allow_anonymous()
+        response.Headers.Location!.ToString().ShouldStartWith("/bff/login");
+        response.Headers.Location!.ToString().ShouldContain("redirectUri=/");
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        response = await BffHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(IdentityServerHost.Url("/connect/authorize"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+
+        await IdentityServerHost.IssueSessionCookieAsync("alice");
+        response = await IdentityServerHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(BffHost.Url("/signin-oidc"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        response = await BffHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldBe("/bff/silent-login-callback");
+    }
+
+    [Fact]
+    public async Task can_also_issue_silent_login_with_prompt()
+    {
+        var response = await BffHost.BrowserClient.GetAsync(BffHost.Url("/bff/login?prompt=none"));
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(IdentityServerHost.Url("/connect/authorize"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        await IdentityServerHost.IssueSessionCookieAsync("alice");
+        response = await IdentityServerHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldStartWith(BffHost.Url("/signin-oidc"));
+        response.Headers.Location!.ToString().ShouldNotContain("error");
+
+        response = await BffHost.BrowserClient.GetAsync(response.Headers.Location.ToString());
+        response.StatusCode.ShouldBe(HttpStatusCode.Redirect);
+        response.Headers.Location!.ToString().ShouldBe("/bff/silent-login-callback");
+    }
+
+    [Fact]
+    public async Task login_with_unsupported_prompt_is_rejected()
+    {
+        var response = await BffHost.BrowserClient.GetAsync(BffHost.Url("/bff/login?prompt=select")); // select is not supported now. 
+        response.StatusCode.ShouldBe(HttpStatusCode.BadRequest);
+
+        var problem = await response.Content.ReadFromJsonAsync<HttpValidationProblemDetails>();
+        problem!.Errors.ShouldContainKey("prompt");
+        problem!.Errors["prompt"].ShouldContain("prompt 'select' is not supported");
+
+
+    }
+
+    [Fact]
+    public async Task can_also_issue_login_with_prompt_select()
```

Comment:
Rename this test - you're testing prompt=create

Reply:
done